server/migrations/0026_merge_0025_event_location_0025_player_imported_data.py

```diff
@@ -0,0 +1,12 @@
+# Generated by Django 4.2.2 on 2023-08-30 12:52
+
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+    dependencies = [
+        ("server", "0025_event_location"),
```

Comment:
Working with merge migrations could be a mess. Can you rollback the migrations to `0024`, and recreate them to create a linear migration dependency?

Reply:
did this and pushed now